### PR TITLE
Pull in upstream updates

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request pulls in upstream updates from [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role).

## 💭 Motivation and Context

I plan to make some changes to the code-gov-update deployment related to cisagov/cyhy_amis#279, so it makes sense to go ahead and update all the related repositories.

## 🧪 Testing

All pre-commit hooks pass.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -3,13 +3,37 @@ galaxy_info:
...
+    - name: Fedora
+      versions:
+        - 32
+        - 33
+    - name: Ubuntu
+      versions:
+        - xenial
+        - bionic
+        - focal
+  role_name: docker
```
Comment (dav3r) [inline meta/main.yml]: `role_name` should be `code_gov_update`.

**Review by dav3r — APPROVED** [review]
Approved after you make one small change. 🐎

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # ansible-role-code-gov-update #
 
-[![Build Status](https://travis-ci.com/cisagov/ansible-role-code-gov-update.svg?branch=develop)](https://travis-ci.com/cisagov/ansible-role-code-gov-update)
+[![GitHub Build Status](https://github.com/cisagov/skeleton-ansible-role/workflows/build/badge.svg)](https://github.com/cisagov/skeleton-ansible-role/actions)
```
Comment (mcdonnnj) [inline README.md]: URL needs to be updated.

File: meta/main.yml (hunk 1 above)
Comment (jsf9k) [reply meta/main.yml]: See commit 804f7c8.

**Review by mcdonnnj — APPROVED** [review]
Thanks for doing the work 💪 Just one change and off we go on my end.

File: README.md (hunk 2 above)
Comment (jsf9k) [reply README.md]: See commit 58b4738.

**Pull request merged**